frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts

```diff
@@ -49,7 +49,7 @@ export class EnrollmentOverTimeComponent implements OnChanges, OnInit, OnDestroy
   graphInfoSub: Subscription;
   isGraphLoading$ = this.experimentService.isGraphLoading$;
 
-  constructor(private experimentService: ExperimentService) { }
+  constructor(private experimentService: ExperimentService) { this.formateXAxisLabel = this.formateXAxisLabel.bind(this) }
```

Comment:
@RidhamShah Can you please add semicolon in the end of statement inside the constructor?

Reply:
code updated